src/spirc.rs

```diff
@@ -378,6 +465,27 @@ impl SpircTask {
         }
     }
 
+    fn handle_play(&mut self) {
```

Comment:
Add `assert_eq!(self.state.get_status(), PlayStatus::kPlayStatusPause)` and document it as a precondition